### PR TITLE
Add verification to balances page

### PR DESCRIPTION
**Pull request by gregnazario**
Adds the same thing to account coin balances as it did to balance changes

### CODE REVIEW

File: src/components/Table/VerifiedCell.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            <WarningAmberOutlined fontSize="small" />
+          </StyledTooltip>
+        </Stack>
+      ) : (
+        <Stack direction="row" spacing={1} alignItems="center">
+          <StyledTooltip title="This asset is not verified, it may or may not be recognized by the community.">
+            <WarningOutlined fontSize="small" />
+          </StyledTooltip>
+        </Stack>
+      )}
```
Comment (kaw2k) [inline src/components/Table/VerifiedCell.tsx:90]: Mind extracting this into a `switch` or something similar. Deeply nested ternaries in render statements are an eye-full.

File: src/pages/Account/Components/CoinsTable.tsx [hunk 2]
```diff
@@ -64,62 +61,79 @@ function CoinTypeCell({
...
           <GeneralTableHeaderCell header="Name" />
+          <GeneralTableHeaderCell header="Asset Type" />
+          <GeneralTableHeaderCell header="Asset" />
+          <GeneralTableHeaderCell
+            header="Verified"
+            tooltip={
+              <LearnMoreTooltip
+                text="This uses the Panora token list to verify authenticity of known assets on-chain.  It does not guarantee anything else about the asset and is not financial advice."
+                link="https://github.com/PanoraExchange/Aptos-Tokens"
+              />
```
Comment (kaw2k) [inline src/pages/Account/Components/CoinsTable.tsx:105]: If you wouldn't mind modifying `LearnMoreTooltip` to add `target="_blank" to the `Link`. That way we open these external links in new tabs.

**Comment by kaw2k** [comment]
Thanks for adding this!

File: src/components/Table/VerifiedCell.tsx (hunk 1 above)
Comment (gregnazario) [reply src/components/Table/VerifiedCell.tsx:90]: Yeah, it truly is, I'm not the greatest at this stuff, let me see what I can do

**Review by kaw2k — APPROVED** [review]

**Pull request merged**